scripts/downlevel-dts/index.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
```

Comment:
An idea: we should use ts-node hashbang for all of our scripts in the future: https://github.com/TypeStrong/ts-node/issues/116

Reply:
Yup, I would also like to use a single yargs instance in `scripts` followed by commands in each subfolder.